app/views/candidate_mailer/tailored_rejection_advice/_teaching_knowledge_tailored_advice.text.erb

```diff
@@ -0,0 +1,13 @@
+<% if @application_choice.tailored_advice_reasons["teaching_knowledge"].present? %>
+  <% if @application_choice.render_tailored_advice_section_headings? %>
+    <% if @application_choice.tailored_advice_reasons['teaching_knowledge'].include? 'subject_knowledge' %>
```

Comment:
Two nested if statements in a view feel like a lot, is there a way we could tidy up this view by using helpers or simplifying the logic?

Reply:
I agree, this is pretty ugly, let me have a look.

Reply:
Turns out those conditions aren't necessary anymore, leftover from an earlier iteration when sometimes the values were nil, so I could just delete it. Glad you flagged it.